MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -650,3 +612,31 @@ extension HomeViewController {
         storeButton.addTarget(self, action: #selector(onStoreModalShow), for: .touchUpInside)
     }
 }
+
+// MARK: Background color behavior
+extension HomeViewController {
+    private func applyBackgroundGradient(duration: TimeInterval) {
+        let locations: [NSNumber] = [0.0, 0.06, 0.18, 0.3, 0.39, 0.51, 0.68, 0.89, 1.0]
+        gradientLayer.locations = locations
+        guard let oldColors = gradientLayer.colors as? [CGColor] else { return}
+        animateBackgroundColors(fromColors: oldColors, toColors: CGColor.mainGradient, duration: duration)
+    }
+
+    private func applyBackgroundNavBar(duration: TimeInterval) {
+        // Temporary solution. Ideally, there should be a custom nav bar so that you do not have to change the color of the controller under the standard nav bar
+        guard let oldColors = gradientLayer.colors as? [CGColor] else { return}
+        animateBackgroundColors(fromColors: oldColors, toColors: [Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor, Theme.shared.colors.navigationBarBackground!.cgColor], duration: duration)
```

Comment:
Can format longer lines to rather use multiple lines, makes the code more readable